XiEditor/EditViewController.swift

```diff
@@ -78,9 +78,14 @@ class EditViewController: NSViewController, EditViewDataSource {
         editView.dataSource = self
         gutterView.dataSource = self
         scrollView.contentView.documentCursor = NSCursor.iBeam();
-        
+    }
+
+    override func viewDidAppear() {
+        super.viewDidAppear()
         NotificationCenter.default.addObserver(self, selector: #selector(EditViewController.boundsDidChangeNotification(_:)), name: NSNotification.Name.NSViewBoundsDidChange, object: scrollView.contentView)
         NotificationCenter.default.addObserver(self, selector: #selector(EditViewController.frameDidChangeNotification(_:)), name: NSNotification.Name.NSViewFrameDidChange, object: scrollView)
+        // call to set initial scroll position once we know view size
+        updateEditViewScroll()
```

Comment:
I'm ok with this change, but in my testing we _were_ getting the notification from Cocoa, it's just that it wouldn't reliably be conveyed to the core because the other initialization might not have completed.